src/rest_framework_dso/fields.py

```diff
@@ -42,8 +42,12 @@ def __init__(self, fields: list[str] | None = None, prefix: str = ""):
         self._children: dict[str, FieldsToDisplay] = {}
         self._allow_all = True
         self.prefix = prefix
+        self._ordered_fields: list[str] = []
 
         if fields:
+            self._ordered_fields = list(
+                filter(lambda f: not f.startswith("-") and "." not in f, fields)
+            )
```

Comment:
Wat gebeurt er met `?_fields=-alleen_een_exclude` ?
```suggestion
            self._ordered_fields = [f for f in fields if not f.startswith("-") and "." not in f]
```